docker-compose.yaml

```diff
@@ -280,9 +281,7 @@ services:
     <<: *inference-service-cpu
     container_name: chat-completions
     profiles: [chat_completions_mistralrs_cpu]
-    build:
-      context: https://github.com/EricLBuehler/mistral.rs.git
-      dockerfile: Dockerfile
+    image: ghcr.io/ericlbuehler/mistral.rs:cpu-0.4
```

Comment:
Do we really want to fetch this image and not build locally (for arm based archs)?